src/Nethermind/Nethermind.Network.Test/P2P/Subprotocols/Eth/StatusMessageSerializerTests.cs

```diff
@@ -51,6 +51,21 @@ public void Roundtrip()
             Assert.AreEqual(statusMessage.ProtocolVersion, deserialized.ProtocolVersion, $"{nameof(deserialized.ProtocolVersion)}");
         }
         
+        [Test]
+        public void Hobbit()
```

Comment:
Why the name?

Reply:
strangely the ETh 2.0 team came up with exactly the same name for their pipeline testing totally independent

it feels so natural to name 'there and back' style tests Hobbit tests (hobbit: there and back again)